hugo/content/configuration-overview.md

```diff
@@ -0,0 +1,242 @@
+The configuration files for the NGINX Agent are `/etc/nginx-agent/nginx-agent.conf` and `/etc/nginx-agent/agent-dynamic.conf`. These files have comments at the top indicating their purpose.
+
+Examples of the configuration files are provided below:
+
+<details open>
+    <summary>example nginx-agent.conf</summary>
+
+{{<note>}}
+In the following example `nginx-agent.conf` file, you can change the `server.host` and `server.grpcPort` to connect to the control/management plane.
+{{</note>}}
+
+```nginx {hl_lines=[13]}
+#
+# /etc/nginx-agent/nginx-agent.conf
+#
+# Configuration file for NGINX Agent.
+#
+# This file tracks agent configuration values that are meant to be statically set. There  
+# are additional NGINX Agent configuration values that are set via the API and agent install script
+# which can be found in /etc/nginx-agent/agent-dynamic.conf. 
+
+# specify the server grpc port to connect to
+server:
+  # host of the control plane
+  host: <FQDN>
+  grpcPort: 443
+# tls options
+tls:
+  # enable tls in the nginx-agent setup for grpcs
+  # default to enable to connect with secure connection but without client cert for mtls
+  enable: true
+  # controls whether the server certificate chain and host name are verified.
+  # for production use, see instructions for configuring TLS
+  skip_verify: false
+log:
+  # set log level (panic, fatal, error, info, debug, trace; default "info")
+  level: info
+  # set log path. if empty, don't log to file.
+  path: /var/log/nginx-agent/
+nginx:
+  # path of NGINX logs to exclude
+  exclude_logs: ""
+  # Set to true when NGINX configuration should contain no warnings when performing a configuration apply (nginx -t is used to carry out this check)
+  treat_warnings_as_errors: false # Default is false
+# data plane status message / 'heartbeat'
+dataplane:
+  status:
+    # poll interval for dataplane status - the frequency the agent will query the dataplane for changes
+    poll_interval: 30s
+    # report interval for dataplane status - the maximum duration to wait before syncing dataplane information if no updates have been observed
+    report_interval: 24h
+metrics:
+  # specify the size of a buffer to build before sending metrics
+  bulk_size: 20
+  # specify metrics poll interval
+  report_interval: 1m
+  collection_interval: 15s
+  mode: aggregated
+
+# OSS NGINX default config path
+# path to aux file dirs can also be added
+config_dirs: "/etc/nginx:/usr/local/etc/nginx"
+
+# Enable reporting NGINX App Protect details to the control plane.
+nginx_app_protect:
+  # Report interval for NGINX App Protect details - the frequency the NGINX Agent checks NGINX App Protect for changes.
+  report_interval: 15s
+  # Enable precompiled publication from the NGINX Management Suite (true) or perform compilation on the data plane host (false).
+  precompiled_publication: true
+```
+
+</details>
+
+
+<details open>
+    <summary>example dynamic-agent.conf</summary>
+
+```yaml
+#
+# /etc/nginx-agent/dynamic-agent.conf
```

Comment:
`/etc/nginx-agent/agent-dynamic.conf` should be `/var/lib/nginx-agent/agent-dynamic.conf`